src/backends/backend/triton_backend_manager.cc

```diff
@@ -224,6 +226,25 @@ TritonBackend::~TritonBackend()
   LOG_STATUS_ERROR(UnloadBackendLibrary(), "failed unloading backend");
 }
 
+void
+TritonBackend::SetUnloadEnabled(const bool enable)
+{
+  // For memory leak debugging do not allow shared libraries to be
```

Comment:
I believe this comment should still be under UnloadBackendLibrary()

Reply:
It refers to the immediately following code. Will update the comment to be more clear.